qt/scientific_interfaces/Muon/ALCPeakFittingView.cpp

```diff
@@ -31,7 +33,7 @@ IPeakFunction_const_sptr ALCPeakFittingView::peakPicker() const { return m_peakP
 void ALCPeakFittingView::initialize() {
   m_ui.setupUi(m_widget);
 
-  connect(m_ui.fit, SIGNAL(clicked()), this, SIGNAL(fitRequested()));
+  connect(m_ui.fit, SIGNAL(clicked()), this, SLOT(fitRequested()));
```

Comment:
Can you implement the connections using the new qt syntax in signals/slots of this class?
```suggestion
  connect(m_ui.fit, &QPushButton::clicked, this, &ALCPeakFittingView::fitRequested);
```